zipline/algorithm.py

```diff
@@ -1026,6 +1029,10 @@ def set_commission(self, commission):
         self.blotter.commission = commission
 
     @api_method
+    def set_cancel_policy(self, cancel_policy):
```

Comment:
@jbredeche re: https://github.com/quantopian/zipline/pull/1017/files#r54516398 it looks like it is here.
I believe the other `cancel_policy` in kwargs setting a default.